utils.js

```diff
@@ -274,6 +274,72 @@ function toBase64(hexString) {
   return Buffer.from(hexString, "hex").toString("base64");
 }
 
+/**
+ * Return the current timestamp in YYYY-MM-DDTHH:MM:SSZ format.
+ *
+ * @returns {string} ISO formatted timestamp, without milliseconds.
+ */
+export function getTimestamp() {
+  return new Date().toISOString().split(".")[0] + "Z";
+}
+
+/**
+ * Method to determine if a license is a valid SPDX license expression
+ *
+ * @param {string} license License string
+ * @returns {boolean} true if the license is a valid SPDX license expression
+ * @see https://spdx.dev/learn/handling-license-info/
+ **/
+export function isSpdxLicenseExpression(license) {
+  const licenseLoweCase = (license || "").toLowerCase();
+  if (!licenseLoweCase) {
```

Comment:
This condition will always be false since we subsititute empty value above.

Reply:
If you receive an `undefined`/`null` value you will have an empty string here. The empty string is "falsy". I was thinking it's not worth it to do all the logic on an empty string.

Reply:
Oh yeah. For some reason, I forgot this falsy thing and assumed only python behaves this way. Another learning for today!